test/e2e_federation/federation-util.go

```diff
@@ -33,7 +33,9 @@ import (
 	"k8s.io/kubernetes/pkg/client/restclient"
 	"k8s.io/kubernetes/pkg/client/unversioned/clientcmd"
 	clientcmdapi "k8s.io/kubernetes/pkg/client/unversioned/clientcmd/api"
+	"k8s.io/kubernetes/test/e2e/common"
 	"k8s.io/kubernetes/test/e2e/framework"
+	fedframework "k8s.io/kubernetes/test/e2e_federation/framework"
```

Comment:
some of the utility functions from e2e.framework package are being used in federation e2e tests like `ExpectNoError`, `Failf`, these are not methods of framework type, so we need to include both these packages e2e.framework and e2e_federation.framework for now.